src/nvim/ex_docmd.c

```diff
@@ -9883,7 +9883,7 @@ static void ex_terminal(exarg_T *eap)
 
 /// Checks if `cmd` is "previewable" (i.e. supported by 'inccommand').
 ///
-/// @param[in] cmd Commandline to check. May start with a range.
+/// @param[in] cmd Commandline to check. May start with a range or modifer.
```

Comment:
Small thing but 'modifer' should be 'modifier'

Reply:
Good catch.  I'll fix that tonight.